x-pack/test/functional_cors/config.ts

```diff
@@ -8,6 +8,7 @@
 import Url from 'url';
 import Path from 'path';
 import type { FtrConfigProviderContext } from '@kbn/test/types/ftr';
+// @ts-ignore
```

Comment:
Why was this needed?

Reply:
Same as the above

Reply:
Maybe we should file an issue with details about the debt we're taking on here and link to it from all the `@ts-expect-error` directives (and I'd prefer those over `@ts-ignore`)

Reply:
I've changed all introduced `@ts-ignore` into `@ts-expect-error` and grouped all the files part of this tech debt at https://github.com/elastic/kibana/issues/95679#issuecomment-810477506

Reply:
Sorry, but can we link the issue along with the directives like:

```
@ts-expect-error https://github.com/elastic/kibana/issues/95679
```

Really want to make it obvious what needs to be done next time we're in this code.

Reply:
Done 😃